Fail to build on RHEL 6

Installation and build are failing due to : 
........
apps_plugin.o: In function `main':
apps_plugin.c:(.text+0x7b44): undefined reference to `CAP_IS_SUPPORTED'
apps_plugin.c:(.text+0x7b68): undefined reference to `CAP_IS_SUPPORTED'
collect2: ld returned 1 exit status
make[2]: *** [apps.plugin] Error 1
make[2]: *** Waiting for unfinished jobs....
mv -f .deps/sys_kernel_mm_ksm.Tpo .deps/sys_kernel_mm_ksm.Po
make[2]: Leaving directory `/home/user/netdata/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/user/netdata'
make: *** [all] Error 2

--- a/src/apps_plugin.c
+++ b/src/apps_plugin.c
@@ -3026,20 +3026,6 @@ static int am_i_running_as_root() {
 
 #ifdef HAVE_CAPABILITY
 static int check_capabilities() {
-    if(!CAP_IS_SUPPORTED(CAP_DAC_READ_SEARCH)) {
-        error("This system does not support CAP_DAC_READ_SEARCH capability. Please setuid to root apps.plugin.");
-        return 0;
-    }
-    else if(debug)
-        info("System has CAP_DAC_READ_SEARCH capability.");
-
-    if(!CAP_IS_SUPPORTED(CAP_SYS_PTRACE)) {
-        error("This system does not support CAP_SYS_PTRACE capability. Please setuid to root apps.plugin.");
-        return 0;
-    }
-    else if(debug)
-        info("System has CAP_SYS_PTRACE capability.");
-
     cap_t caps = cap_get_proc();
     if(!caps) {
         error("Cannot get current capabilities.");

